### PR TITLE
Use developer's stack to run integrations tests - [CUMULUS-615]

### PR DESCRIPTION
**Pull request by scisco**
**Summary:** Summary of changes

Addresses [CUMULUS-615](https://bugs.earthdata.nasa.gov/browse/CUMULUS-615)

## Changes

* Run integration tests on stack associated with the committer

## Test Plan
Things that should succeed before merging.

- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Update CHANGELOG
- [ ] Run `./bin/eslint-ratchet` and verify that eslint errors have not increased
  - [ ] Commit `.eslint-ratchet-high-water-mark` if the score has improved

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -17,22 +17,12 @@ These tests run against AWS, so a Cumulus deployment is needed. Set up the deplo
...
-Configuration can be overriden in your own `spec/config.override.yml`. If you are getting setup for the first time:
-
-```
-cp spec/config.yml spec/config.override.yml
-```
-
-And then edit `spec/config.override.yml`.
-
-Using an override file is required if using a stack other than the `test-cumulus` stack in the `cumulus-sndbx` AWS account. If you want to switch back to the default `spec/config.yml` file, you can specify `USE_DEFAULT_CONFIG=true` when running tests. E.g.:
+Use the name of your deployment to run the tests by setting the `DEPLOYMENT` environment variable. For example:
```
Comment (laurenfrederick) [inline example/README.md]: I think we still need the part about copying and updating the override file

File: example/spec/select [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+if [ ! -z "${CIRCLE_USERNAME}" ]; then
+    declare -A developers
+    developers=( ["abarciauskas-bgse"]="aimee-test" \
```
Comment (laurenfrederick) [inline example/spec/select]: Should we put something in the README about this?

Comment (scisco) [reply example/spec/select]: yeah, we can.

File: example/README.md [hunk 3]
```diff
@@ -63,19 +53,19 @@ To access test data in `s3://cumulus-data-shared`, which is required by all spec
 
 Tests are written and run with [jasmine](https://jasmine.github.io/setup/nodejs.html).
 
-To run all of the tests, run `npm test` in the top level of the repository.
+To run all of the tests, run `DEPLOYMENT=<name-of-your-deployment> npm test` in the top level of the repository.
```
[the remark this answers is not in the document]
Comment (scisco) [reply example/README.md]: actually no. I'm going to update the README. as long as it is defined in the `.env` file which is required for the deployment to work, it should work fine.

**Pull request merged**